spec/dummy/db/migrate/20180917164000_create_action_mailbox_tables.rb

```diff
@@ -0,0 +1,19 @@
+class CreateActionMailboxTables < ActiveRecord::Migration[6.0]
```

Comment:
Should this be version 7?

Reply:
I think the problem was it melting supporting non-Rails 7 versions or something, it doesn't really matter

Reply:
Yeah. But I don't think we have to change the version anyway, like it works.